angular-rails-app/app/assets/javascripts/property/property.html

```diff
@@ -131,6 +134,15 @@ <h4 class="table-title">Open HPD Violations</h4>
                 <td>{{ v.certified_date }}</td>
               </tr>
             </table>
+            <div class="text-center">
+                <dir-pagination-controls
+                pagination-id="hpd_violations"
+                on-page-change="vm.hpdViolationsPageChanged(newPageNumber)"
+                max-size="10"
+                direction-links="true"
+                boundary-links="true" >
+              </dir-pagination-controls>
+            </div>
```

Comment:
Ya, definitely agree with the refactor.

One other thing you could do to make the templates cleaner would be to run the HTML through a beautifier. [A quick google turns up a bunch of different tools.](https://www.google.com/webhp?sourceid=chrome-instant&rlz=1C5CHFA_enUS503US504&ion=1&espv=2&ie=UTF-8#q=html+beautifier)